.changeset/fresh-turkeys-grow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hyperlane-xyz/registry': patch
+---
+
+Improve CI workflow by separating concerns
```

Comment:
This is totally fine but FYI, changes that don't affect the published data don't need a changeset. Yes the changeset CI will fail but that's expected.